### PR TITLE
feat(spans): collect exclusive_time_light for cache spans

### PR DESCRIPTION
**Pull request by DominikB2014**
1. Collect exclusive time light metrics for cache.* spans
- We have cases where we don't need to filter by transaction in the UI and without the light metric, discover will not return any data. Discover always attemps to query for the light metric if the transaction field is not passed into it in the query.

2. Fix some typo's, we were trying to extract `cache_hit` instead of `cache.hit` as defined here
https://github.com/getsentry/relay/blob/72918c1be88e4a034fa807ca03b86f57e23097c2/relay-event-normalization/src/normalize/span/tag_extraction.rs#L108

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
```
Comment (DominikB2014) [inline CHANGELOG.md:3]: @getsentry/ingest is there anything wrong with my changelog entry here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+**Features**:
+
+**Internal**:
+
+- Collect exclusive_time_light metrics for cache.\* spans ([#3466]https://github.com/getsentry/relay/pull/3466)
```
Comment (phacops) [inline CHANGELOG.md]:
```suggestion
- Collect exclusive_time_light metrics for `cache.*` spans. ([#3466](https://github.com/getsentry/relay/pull/3466))
```

File: CHANGELOG.md (hunk 1 above)
Comment (phacops) [reply CHANGELOG.md:3]: Yes, see below for my suggestion.

File: CHANGELOG.md (hunk 2 above)
Comment (colin-sentry) [inline CHANGELOG.md]: the format is incorrect, should be

`([#3466](https://github.com/getsentry/relay/pull/3466))`

**Review by phacops — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (DominikB2014) [reply CHANGELOG.md]: Thanks!

**Review by jjbayer — APPROVED** [review]

**Review by iker-barriocanal — APPROVED** [review]

**Pull request merged**